sway/commands/resize.c

```diff
@@ -292,6 +292,11 @@ static struct cmd_results *resize_adjust_tiled(uint32_t axis,
  */
 static struct cmd_results *resize_set_tiled(struct sway_container *con,
 		struct movement_amount *width, struct movement_amount *height) {
+	if (container_is_scratchpad_hidden_or_child(con)) {
+		return cmd_results_new(CMD_FAILURE,
+			"Cannot resize a hidden scratchpad or child container");
+	  }
```

Comment:
Style: 3the codebase uses tabs, but this patch uses mixed spaces and tabs.

Reply:
Hi @emersion,
thanks, this slipped by. Fixed in the last force-push.